apps/web/src/pages/blocks.tsx

```diff
@@ -174,6 +190,24 @@ const Blocks: NextPage = function () {
 
           return {
             cells: [
+              {
+                item: (
+                  <div className="relative flex">
+                    {transactions.map((tx, i) => {
+                      return tx.rollup ? (
+                        <div
+                          key={i}
+                          className="-ml-1 -translate-y-1 scale-100 first-of-type:ml-0 hover:scale-125 "
```

Comment:
I'd prefer to remove the scale animation to keep consistency with the rest of tables

Reply:
Done